src/commands/games/mcquery.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-control-regex */
+const { CommandStructures, SwitchbladeEmbed, Constants } = require('../../')
+const { Command, CommandParameters, StringParameter } = CommandStructures
+
+module.exports = class MCQuery extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'mcquery'
+    this.category = 'games'
+    this.aliases = ['minecraftquery']
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ missingError: 'commands:mcquery.noIP' })
+    )
+  }
+  async run ({ t, author, channel }, server) {
+    let address = server.split(':')
+    let port = 25565
+    if (address.length === 2) port = address[1]
```

Comment:
I think lines 16, 17 and 18 should be replaced with
```js
let [ address, port = 25565 ] = server.split(':');
```
and then you would use just `address` and `port` instead of `address[0]` or `address[1]`.